utilities/xmlrpcpp/cmake/xmlrpcpp-extras.cmake.in

```diff
@@ -0,0 +1 @@
+list(APPEND @PROJECT_NAME@_INCLUDE_DIRS ${@PROJECT_NAME@_DIR}/../../../@CATKIN_GLOBAL_INCLUDE_DESTINATION@/xmlrpcpp)
```

Comment:
The value should be enclosed in quotes - just in case some part of the path contains spaces.